MonstieWash/Assets/Scripts/Animation Control/MonsterAnimControl.cs

```diff
@@ -3,35 +3,27 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-[Serializable]
-public class MoodToAnimName
-{
-    public string moodName;
-    public string animName;
-}
-
 public class MonsterAnimControl : MonoBehaviour
 {
     private MonsterBrain m_mimicAI;   
```

Comment:
One small thing I missed is to rename this variable since this is intended as a general animation controller for all monsters not just the mimic.
```suggestion
    private MonsterBrain m_monsterAI;   
```